pipelines/nextflow/modules/gff3/process_gff3.nf

```diff
@@ -17,19 +17,21 @@ nextflow.enable.dsl=2
 params.merge_split_genes="False"
 
 process PROCESS_GFF3 {
-    tag "$gff3 - $task.attempt"
+    tag "$gca"
     label 'variable_2_8_32'
-    errorStrategy 'ignore'
+    errorStrategy 'finish'
```

Comment:
Does this mean, it will not proceed with other species but allow the already run jobs to finish in case of an error?

Reply:
Maybe I shouldn't include this here; but this means that it will allow the jobs already running to finish. Ignore would continue running all jobs, even the ones after that one (and we don't want that).

Reply:
so if a species enquires an error then the other species will just continue the job that is already running but not proceed further? this can lead to an incomplete run for other species? but alternatively can be resumes?

Reply:
Yes, basically: let the pipeline finish what it's already started, fix the issue for the gff3 that failed, and resume

Reply:
okay seems good to me then, better to resume then missing the error